### PR TITLE
Extras and rearrange

### PR DESCRIPTION
**Pull request by autodidaddict**
When you're looking at the diff it looks like I've gotten the names of the schemas wrong in the `codegen.yaml` files... it's just the differ getting a little confused. The actual files should be accurate.

### CODE REVIEW

File: extras/rust/src/lib.rs [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+//! This crate provides wasmCloud actors with an interface to the extras capability provider.
+//! Every wasmCloud host runtime automatically comes with a built-in extras provider. However,
+//! actors using this provider will still need to be signed with the `wasmcloud:extras`
+//! capability contract ID.
+//!
+//! The following functions are supported on the extras `Host` interface:
+//!
+//! * [request_guid](generated::Host::request_guid)
+//! * [request_sequence](generated::Host::request_sequence)
+//! * [request_random](generated::Host::request_random)
```
Comment (brooksmtownsend) [inline extras/rust/src/lib.rs:12]: Do we want to include a rustdoc test to show what it looks like to use these functions?

**Review by brooksmtownsend — CHANGES_REQUESTED** [review]
Looks good to me, I like the pattern of including rustdoc tests for these actor-interfaces to show the "real-world" example. So my only request is to have a simple actor example shown in the rustdoc to show people what interface they need to import + how to call these functions

**Review by billyoung — APPROVED** [review]
looks good!

File: extras/rust/src/lib.rs (hunk 1 above)
Comment (autodidaddict) [reply extras/rust/src/lib.rs:12]: Yes, I'll actually get that into the next forthcoming PR. I switched branches too early.

**Pull request merged**